### ISSUE TITLE
Getting payara tools timeout exception when domain has secure admin enabled

### ISSUE THREAD

**Issue by dkechrologos** [payara/ecosystem-eclipse-plugin#22]
Getting payara tools timeout exception when domain has secure admin enabled and i use password in properites page. If domain has no secure admin and I don't enter a pass, it starts up ok. E.g. when trying default "domain1".

![image](https://user-images.githubusercontent.com/54798055/73617505-1d27b880-4628-11ea-9760-28f57900564e.png)

In eclipse log I am getting this:

!ENTRY org.eclipse.payara.tools 1 1 2020-02-03 01:52:27.639
!MESSAGE Payara: ServerStatusMonitor for Payara Server 5 (5.194) [dk.domains] location throws exception

**Comment by jGauravGupta** [comment]
Hi @dkechrologos,

I found a similar opened ticket ECOSYS-33 [Prompt for username/password when credentials incorrect upon Payara startup].

> This happens when the domain has secure admin enabled and I use the password in the properties page.

Does timeout happen even a correct credential is entered in the properties page?

**Comment by dkechrologos** [comment]
Yes, even when correct credentials are entered. Before timeout occures I can open a browser and login to admin consol from there!
If I enter **incorrect** credentials to payara tools, a pop up tells me that wrong username or password enetered and shuts doewn the server.

**Comment by jGauravGupta** [comment]
Thanks @dkechrologos,

Created [ECOSYS-109](https://payara.atlassian.net/browse/ECOSYS-109) to address the issue.
